coalib/testing/LocalBearTestHelper.py

```diff
@@ -113,7 +117,7 @@ def check_results(self,
                               list,
                               msg="The given results are not a list.")
 
-        with prepare_file(lines, filename,
+        with prepare_file(lines, filename, settings,
```

Comment:
settings should go into `execute_bear`, this is the wrong function :) This one prepares a temporary file with the contents given by `filename`, so has nothing to do with bear execution^^